features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Description
+
+This document describes how to update the mbed OS LWIP stack. The mbed OS LWIP stack is a copy of the LWIP master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
+
+When you add new releases or single commits from the LWIP master repository, you must fetch them using the mbed OS LWIP fork repository. Use the repository to rename LWIP source files with the `lwip_` prefix to make them compatible with the mbed OS build system.
```

Comment:
Please change "mbed" to "Mbed" both times in this line to reflect the new branding rollout.